### ISSUE TITLE
[Bug]: Invalid registry key path in windows.yaml

### ISSUE THREAD

**Issue by megamurmulis** [undergroundwires/privacy.sexy#446]
### Description

```
FILE: \src\application\collections\windows.yaml
LINE: 481
OLD:  keyPath: HKCU\Software\Microsoft\Windows\v
FIX:  keyPath: HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\WordWheelQuery
```
`Importance:` Low (as registry key not used in newer Windows versions)

Path incorrect ~ likely due not applying sufficient force on [`CTRL`] key when pasting (`CTRL+v`).

### Reproduction steps

N/A

### Expected behavior

Fixed registry key path (`HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\WordWheelQuery`)

**Comment by megamurmulis** [comment]
Also wrong valueName:
```
  name: Disable Automatic Updates (AU) feature
  valueName: Enabled
  data: "1"
```
=> probably should be:
`  valueName: NoAutoUpdate`

**Comment by undergroundwires** [comment]
Thank you @megamurmulis, I've fixed those and they'll be released as part of next release, please report more issues in new threads if you encounter any.

**Comment by undergroundwires** [comment]
Fix released as part of [0.13.7](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.13.7) 🚀. Feel free to test it and open new issue if there are any other issues. Thank you again.
